tests/handlers/test_password_providers.py

```diff
@@ -439,7 +535,7 @@ def _send_password_login(self, user: str, password: str) -> FakeChannel:
         return self._send_login(type="m.login.password", user=user, password=password)
 
     def _send_login(self, type, user, **params) -> FakeChannel:
-        params.update({"user": user, "type": type})
+        params.update({"identifier": {"type": "m.id.user", "user": user}, "type": type})
```

Comment:
the old way would have worked too, but we should use the new way where possible. Grandfathering of the old way is tested in `test_grandfathered_identifier`.